Src/BootCamp.Chapter/PeopleAndBalanceManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace BootCamp.Chapter
+{
+    public class PeopleAndBalanceManager
+    {
+        private PersonAndBalance[] _peopleAndBalances;
+        private AnswerFormatter _answerFormatter;
+        public PeopleAndBalanceManager(string[] peopleAndBalances)
+        {
+            _peopleAndBalances = ArrayHandler.CreatePersonsArray(peopleAndBalances);
+            _answerFormatter = new AnswerFormatter();
+        }
+
+        public string GetHighestBalanceEverAnswer()
+        {
+            return _answerFormatter.GetFormattedAnswerForHighestBalanceEver(_peopleAndBalances, FindHighestBalanceEver());
+        }
+
+        public string GetPersonWithBiggestLossAnswer()
+        {
+            if (!isValidForCheck())
+            {
+                return "N/A.";
+            }
+            return _answerFormatter.GetFormattedAnswerForPersonWithBiggestLoss(FindPersonWithBiggestLoss());
+        }
+
+        public string GetRichestPersonAnswer()
+        {
+            var richestPersonsBalance = FindRichestPerson().GetCurrentBalance();
+            return _answerFormatter.GetFormattedAnswerForRichestPerson(_peopleAndBalances, richestPersonsBalance);
+        }
+
+        public string GetPoorestPersonAnswer()
+        {
+            var poorestPersonsBalance = FindPoorestPerson().GetCurrentBalance();
+            return _answerFormatter.GetFormattedAnswerForPoorestPerson(_peopleAndBalances, poorestPersonsBalance);
+        }
+
+        private bool isValidForCheck()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'isValidForCheck' to match pascal case naming rules, consider using 'IsValidForCheck'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)